packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -172,13 +222,60 @@ export class StarkDropdownComponent extends AbstractStarkUiComponent implements
 		if (changes["multiSelect"]) {
 			this.isMultiSelectEnabled = StarkComponentUtil.isInputEnabled(this.multiSelect);
 		}
+
+		if (changes["isDisabled"]) {
+			if (typeof this.dropdownFormControl !== "undefined") {
+				this.logger.warn(reactiveWarningMessage);
+			}
+
+			if (typeof this.formControl !== "undefined") {
+				if (this.isDisabled) {
+					this.formControl.disable();
+				} else {
+					this.formControl.enable();
+				}
+			}
+		}
+
+		if (changes["required"]) {
+			this.setDefaultBlank();
+			if (
+				!changes["required"].firstChange &&
+				typeof this.dropdownFormControl === "undefined" &&
+				typeof this.formControl !== "undefined"
+			) {
+				if (this.required) {
+					this.formControl.setValidators([Validators.required]);
+				} else {
+					this.formControl.clearValidators();
```

Comment:
I don't think we should remove all validators... it could be that the developer added more validators to the form control... we should find a way to remove just the "required" validator but still keep the other ones...

Reply:
In this case, we have the control on the `formControl`, so we can actually do it. 
In any cases, Angular does not provide a way to remove only one validator. It is all or nothing.
You can see this StackOverFlow post about this (they provide a kind of a solution): https://stackoverflow.com/questions/46488078/angular-4-remove-required-validator-conditionally 

If the developer defines a dropdownFormControl, the logic to add/remove the `required` validator to the `formControl` object is not done. That means we can maintain the logic as it is until we need to play with more than one validator internally.

Reply:
Ah true, this logic is only when the developer does not provide a form control... my bad.

About the `clearValidators; indeed you're right, Angular does not provide such feature but for the moment we don't need it :p